mbuild/lib/recipes/alkane.py

```diff
@@ -11,33 +11,29 @@ def __init__(self, n=3, cap_front=True, cap_end=True):
 
         Args:
             n: Number of carbon atoms.
-            cap_front: Add methyl group to beginning of chain ('down' port).
-            cap_end: Add methyl group to end of chain ('up' port).
+            cap_front: Add a hydrogen to the beginning of chain ('down' port).
+            cap_end: Add a hydrogen to the end of chain ('up' port).
         """
-        if n < 2:
+        if n < 1:
             raise ValueError('n must be 1 or more')
         super(Alkane, self).__init__()
 
         # Adjust length of Polmyer for absence of methyl terminations.
-        if not cap_front:
-            n += 1
-        if not cap_end:
-            n += 1
-        chain = mb.recipes.Polymer(CH2(), n=n-2, port_labels=('up', 'down'))
+        chain = mb.lib.recipes.Polymer(CH2(), n=n, port_labels=('up', 'down'))
         self.add(chain, 'chain')
 
         if cap_front:
-            self.add(CH3(), "methyl_front")
+            self.add(H(), "hydrogen_front")
```

Comment:
My only concern is that the hydrogen is not in the same hierarchy as everything else. In addition, the `CH3`s are named `CH2`, which would mess up the residue names in the `to_parmed` conversion as well as the CG examples. 

```python3
>>> alk = mb.recipes.Alkane(2)
>>> print(alk.children)
OrderedSet([<Polymer 6 particles, non-periodic, 5 bonds, id: 4734853456>, <H 1 particles, non-periodic, 0 bonds, id: 4734803152>, <H 1 particles, non-periodic, 0 bonds, id: 4734803728>])
>>> for child in alk.children:
...     print(child)
...     for childchild in child.children:
...             print('\t'+str(childchild))
... 
<Polymer 6 particles, non-periodic, 5 bonds, id: 4734853456>
	<CH2 3 particles, non-periodic, 2 bonds, id: 4734855888>
	<CH2 3 particles, non-periodic, 2 bonds, id: 4734853648>
<H 1 particles, non-periodic, 0 bonds, id: 4734803152>
	<H pos=( 0.0000, 0.0000, 0.0000), 0 bonds, id: 4734803024>
<H 1 particles, non-periodic, 0 bonds, id: 4734803728>
	<H pos=(-0.0000, 0.4200,-0.0000), 0 bonds, id: 4734795344>
```

Reply:
I will try to find a way to fix this issue